docs/integrations/destinations/mssql.md

```diff
@@ -116,6 +116,7 @@ Using this feature requires additional configuration, when creating the source.
 
 | Version | Date       | Pull Request                                               | Subject                                                                                             |
 |:--------|:-----------|:-----------------------------------------------------------|:----------------------------------------------------------------------------------------------------|
+| 0.1.24  | 2023-06-05 | [\#27034](https://github.com/airbytehq/airbyte/pull/27034) | Internal scaffolding update                                                                         |
```

Comment:
Maybe we should have the same release note message: `Internal code change for future development (install normalization packages inside connector)`